### PR TITLE
Use updates instead of full Channel saves(), only update when there a…

### PR DESCRIPTION
**Pull request by nicpottier**
…re changes

### CODE REVIEW

File: temba/channels/views.py [hunk 1]
```diff
@@ -655,9 +655,9 @@ def sync(request, channel_id):
         return JsonResponse({"error_id": 1, "error": "Invalid signature: \'%(request)s\'"
                                                      % {'request': request_signature}, "cmds": []}, status=401)
 
-    # update our last seen on our channel
-    channel.last_seen = timezone.now()
-    channel.save()
+    # update our last seen on our channel if we haven't seen this channel in a bit
+    if not channel.last_seen or timezone.now() - channel.last_seen > timedelta(minutes=5):
+        Channel.objects.filter(id=channel.id).update(last_seen=timezone.now())
```
Comment (rowanseymour) [inline temba/channels/views.py]: Isn't this the same as `channel.save(update_fields=('last_seen',))` ?

Comment (rowanseymour) [inline temba/channels/views.py]: (Which is preferable because it keeps the local object in sync with the db changes)

Comment (nicpottier) [inline temba/channels/views.py]: But that's two lines! :)  But ya, agreed, will change.

**Comment by nicpottier** [comment]
rereview and mege?

**Comment by rowanseymour** [comment]
meged

**Pull request merged**